docs/background/glossary.md

```diff
@@ -11,12 +11,14 @@ Please note that this Glossary is **work in progress**. Do you encounter missing
 
 * cap site: first nucleotide of a transcript (5' end) to which a specially altered nucleotide is added.
 
-* break point: the site where two sequences which are in different positions in the reference sequence are joined as a consequence of genomic rearrangement (Structural Variant)
+* break point: the site where two sequences which are in different positions in the reference sequence are joined as a consequence of genomic rearrangement
 
 * cDNA: cDNA, "copy DNA" or "complementary DNA", is the DNA copy of a single stranded RNA molecule synthesized using the enzyme reverse transcriptase ([Wikipedia](https://en.wikipedia.org/wiki/Complementary_DNA), [MESH](https://www.ncbi.nlm.nih.gov/mesh/68018076)).: **NOTE:** cDNA is not the same as "coding DNA" (see below).
 
 * CDS: coding DNA sequence, a sequence translated in to an amino acid sequence (protein).
 
+* chimeric transcript: a transcript (RNA molecule) composed of transcript segments from two or more genes.
```

Comment:
per @jtdendunnen:
> I see now "chimeric transcript", while I though we decided to use "adjoined transcript".  I do see adjoined transcript mentioned on other pages but it is missing from the Glossary. We now seem to use a mix of chimeric and adjoined. I suggest to use one term only.

Reply:
Adjoined transcripts and chimeric transcripts are distinct concepts (all chimeric transcripts are adjoined transcripts, but not _vice versa_). I think part of what we wanted to accomplish was to include the definition of gene fusions in the glossary, for which we need chimeric transcripts. I agree we should include adjoined transcripts in the glossary as well, but I think we will need both those and chimeric transcripts if we still want to define gene fusions in the HGVS glossary. 

@jtdendunnen, please let me know if you would prefer the definition of gene fusions to be left entirely to the domain of the VICC spec (i.e. removed from the HGVS glossary). FWIW I think it would be a benefit to keep these aligned concepts in the HGVS glossary as well.

Reply:
Addressed in 11797ed. Added an `adjoined transcript` definition and clarified relation between this and `chimeric transcript` in glossary.